opslevel/terraform_type_conversions.go

```diff
@@ -65,11 +65,17 @@ func RequiredIntValue(value int) basetypes.Int64Value {
 }
 
 // Returns value wrapped in a types.StringValue, or types.ListNull if blank
-func OptionalStringListValue(ctx context.Context, value []string) (basetypes.ListValue, diag.Diagnostics) {
-	if len(value) == 0 {
-		return types.ListNull(types.StringType), diag.Diagnostics{}
+func OptionalStringListValue(values []string) basetypes.ListValue {
+	if len(values) == 0 {
+		return types.ListNull(types.StringType)
 	}
-	return types.ListValueFrom(ctx, types.StringType, value)
+
+	elems := make([]attr.Value, len(values))
+	for i, v := range values {
+		elems[i] = types.StringValue(v)
+	}
+
+	return types.ListValueMust(types.StringType, elems)
```

Comment:
Does this just `panic` then?

Reply:
There is a panic but not one we would worry about 😁 

[Under the hood](https://github.com/hashicorp/terraform-plugin-framework/blob/main/types/basetypes/list_value.go#L72-L83) the panic would be thrown if the type of an element in `elems` does not match `types.StringType`. But this is covered at line 75